apps/login/mock/initial-stubs/zitadel.settings.v2alpha.SettingsService.json

```diff
@@ -7,11 +7,40 @@
   {
     "service": "zitadel.settings.v2alpha.SettingsService",
     "method": "GetLegalAndSupportSettings",
-    "out": {}
+    "out": {
+      "data": {
+        "settings": {
+          "tosLink": "http://whatever.com/help",
+          "privacyPolicyLink": "http://whatever.com/help",
+          "helpLink": "http://whatever.com/help"
+        }
+      }
+    }
   },
   {
     "service": "zitadel.settings.v2alpha.SettingsService",
     "method": "GetPasswordComplexitySettings",
-    "out": {}
+    "out": {
+      "data": {
+        "settings": {
+          "minLength": 8,
+          "requiresUppercase": true,
+          "requiresLowercase": true,
+          "requiresNumber": true,
+          "requiresSymbol": true
+        }
+      }
+    }
+  },
+  {
+    "service": "zitadel.settings.v2alpha.SettingsService",
+    "method": "GetLoginSettings",
+    "out": {
+      "data": {
+        "settings": {
+          "passkeysType": 1
+        }
+      }
```

Comment:
Should we really initialize the stubs with concrete values?
I'm afraid of implicit hard dependencies if tests rely on the exact values.

Reply:
I agree for the loginsettings. The rest could stay in there IMHO as the are single use

Reply:
done